### PR TITLE
Fixes issues with butts, reorganizes the module

### PR DESCRIPTION
**Pull request by tortellinitony**
:cl: Tortellini Tony
fix: You can now remove items from butts, and they drop items when removed or destroyed.
/:cl:

Reorganized the butt code so it is no longer lumped together in a module, and is instead where it should be.

### CODE REVIEW

**Comment by GuyonBroadway** [comment]
What an _ass_ pr... Jokes aside I'll review it once I get to my main pc.

File: hippiestation/code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -102,9 +106,6 @@
 			playsound(user, fartsound, 50, 1, 5)
 		sleep(1)
 		if(lose_butt)
-			for(var/obj/item/I in theinv.contents)
-				theinv.remove_from_storage(I, user.loc)
-			B.loc = get_turf(user)
 			B.Remove(user)
```
Comment (Carbonhell) [inline hippiestation/code/modules/mob/living/carbon/human/emote.dm:109]: does removing put the item on the loc by default or what?

File: hippiestation/code/modules/surgery/organs/butts.dm [hunk 2]
```diff
@@ -86,34 +90,13 @@
 
 /obj/item/organ/butt/throw_impact(atom/hit_atom)
 	..()
-	var/mob/living/carbon/M = hit_atom
 	playsound(src, 'hippiestation/sound/effects/fart.ogg', 50, 1, 5)
-	if((ishuman(hit_atom)))
-		M.apply_damage(5, STAMINA)
-		if(prob(5))
-			M.Knockdown(60)
-			visible_message("<span class='danger'>The [src.name] smacks [M] right in the face!</span>", 3)
```
Comment (Carbonhell) [inline hippiestation/code/modules/surgery/organs/butts.dm:95]: why removing this code?

**Review by Carbonhell — DISMISSED** [review]

File: hippiestation.dme [hunk 3]
```diff
@@ -14,6 +14,7 @@
 
 // BEGIN_INCLUDE
 #include "_maps\basemaphippie.dm"
+#include "_maps\hippiestation.dm"
```
Comment (Carbonhell) [inline hippiestation.dme]: wut. this wasn't checked before?the fuck?

File: code/modules/surgery/organs/organ_internal.dm [hunk 4]
```diff
@@ -27,7 +27,7 @@
 	owner = M
 	M.internal_organs |= src
 	M.internal_organs_slot[slot] = src
-	loc = M
+	loc = null
```
Comment (Carbonhell) [inline code/modules/surgery/organs/organ_internal.dm:30]: Are you sure this won't break stuff?I remember it was required for shit like bombs in butts.
Although, there's a PR currently up on TG to give organs a locations, so..

Comment (tortellinitony) [reply code/modules/surgery/organs/organ_internal.dm:30]: yeah wait for that PR for bombs in butts

File: hippiestation.dme (hunk 3 above)
Comment (j-mie) [reply hippiestation.dme]: Remove this, it's forces the hippiestation map ALWAYS and doesn't allow us to pick other maps.

Comment (tortellinitony) [reply hippiestation.dme]: I can uncheck it if necessary

File: hippiestation/code/modules/surgery/organs/butts.dm (hunk 2 above)
Comment (tortellinitony) [reply hippiestation/code/modules/surgery/organs/butts.dm:95]: it doesn't work for some reason

File: hippiestation/code/modules/mob/living/carbon/human/emote.dm (hunk 1 above)
Comment (tortellinitony) [reply hippiestation/code/modules/mob/living/carbon/human/emote.dm:109]: yes

File: hippiestation/code/modules/surgery/organs/butts.dm (hunk 2 above)
Comment (Carbonhell) [reply hippiestation/code/modules/surgery/organs/butts.dm:95]: The only issue i see is that 3 as second arg of visible message

**Review by j-mie — APPROVED** [review]

**Pull request merged**